install_cli.php

```diff
@@ -118,7 +139,7 @@
             echo "DONE";
         }
 
-        echo "\n\tHitting DB... ";
+        echo "\n\tHitting DB... \n";
```

Comment:
Same as below, don't need to add "\n" cause new line will be added to "\n\tWill create.... \n" message